src/platform/unittest/TlsTest.cpp

```diff
@@ -306,6 +306,7 @@ struct TlsTest : public ::testing::TestWithParam<bool>
             uint16_t TPLen = 64
             )
         {
+            ASSERT_NE(nullptr, SecConfiguration);
```

Comment:
Ideally, these checks should go closer to the place the `SecConfiguration` is (attempted to be) created.

Reply:
updated.